src/ProjectTemplates/Web.ProjectTemplates/content/Worker-FSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Worker Service",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1025"
+  },
+  "description": {
+    "text": "An empty project template for creating a worker service.",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
```

Comment:
I just copied this GUID from the C# template, but if we need a new one I'll get a hot one from hotguids: http://www.secretgeek.net/hotGuids/index.htm